Components/Overlay/src/OgreOverlayManager.cpp

```diff
@@ -237,6 +237,12 @@ namespace Ogre {
                     }
                     else
                     {
+                        // Check that the { is not on the same line
+		        if (line[line.length() - 1] == '{')
+                        {
+                            LogManager::getSingleton().logWarning(
```

Comment:
looking at the following code, I think we can just fix the issue instead of warning about it here.
You just need to strip "{" from the name and skip the `skipToNextOpenBrace` call.

Reply:
Assuming that no one actually wants such a name, yes. Also, the possible whitespace between the name and '{' should probably be removed, if there is any (unless that is already handled in Overlay creation).